server/apps/poller/management/commands/poll_maxemail.py

```diff
@@ -130,7 +124,7 @@ def handle(self, *args, **options):
 
         if run_forever:
             while True:
-                self.write("Polling activity stream")
+                self.write("Polling maxemail")
                 self.run(args, options)
                 self.write(f"sleeping until {datetime.now() + timedelta(seconds=60)}")
```

Comment:
another screw up by me: 
```suggestion
                self.write(f"sleeping until {datetime.now() + timedelta(seconds=sleep_time)}")
```